Submitting Evaluation of Jupyter

emmetthannigan opened the pull request:
All questions answered for Jupyter

joannakl commented on jupyter_evaluation.md:
```diff
@@ -1,154 +1,168 @@
--->
 
-1. Project website/homepage:
+Project Jupyter exists to develop open-source software, open-standards, and services for interactive computing. "Jupyter Notebook" is an open-source web application that allows you to create and share documents that contain live code, equations, visualizations and narrative text. "Jupyter Hub" is a multi-user version of the notebook designed for companies, classrooms, and research labs. 
```
If you indent this, the numbering will continue below. Otherwise, the list numbering restarts from 1 again. 
Similar issues below.

emmetthannigan commented:
just added commit "added evaluation_template.md and fixed numbering" which should fix both the numbering and the accidental evaluation_template.md removal
